### PR TITLE
Tags

### PR DESCRIPTION
**Pull request by gcolson**

### CODE REVIEW

File: lib/siilar/client/tags.rb [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      # Edit a tag collection
+      #
+      # @see http://api.siilar.com/1.0/doc/tags#edit-tag-collections
+      def edit_tag_collection(collection, attributes = {})
+        response = client.patch("1.0/tag-collections/#{collection}", attributes)
+
+        Struct::TagCollection.new(response)
+      end
+
+      # Edit a tag collection
```
Comment (mehlah) [inline lib/siilar/client/tags.rb]: s/Edit/Delete ?

**Comment by mehlah** [comment]
Looks good to me! Thanks @gcolson 
Can you please rebase it to remove the merge commit?

**Comment by gcolson** [comment]
is it ok now ?

**Comment by mehlah** [comment]
https://github.com/craftsmen/niland-siilar-ruby/pull/7#discussion_r36284435 still could be corrected. Can you fix it?
Otherwise good job :+1: 
I can release a new version as soon as this got merged.

**Comment by gcolson** [comment]
Do you want me to merge it ?

**Comment by mehlah** [comment]
Should be good!
Thanks

**Pull request merged**